docs/pages/access-controls/sso/gitlab.mdx

```diff
@@ -4,60 +4,91 @@ description: How to configure Teleport access using GitLab for SSO
 h1: Teleport SSO Authentication with GitLab
 ---
 
-## How to use GitLab as a single sign-on (SSO) provider with Teleport
-
 This guide will cover how to configure [GitLab](https://www.gitlab.com/) to issue
-SSH credentials to specific groups of users. When used in combination with role
+credentials to specific groups of users. When used in combination with role
 based access control (RBAC), it allows administrators to define policies
 like:
 
-- Only members of "DBA" group can SSH into machines running PostgreSQL.
+- Only members of "DBA" group can access PostgreSQL databases.
 - Only members of "ProductionKubernetes" can access production Kubernetes clusters
 - Developers must never SSH into production servers.
 
 ## Prerequisites
 
-- At least two groups in GitLab with users assigned.
-- Teleport role with access to maintaining `oidc` resources.  This is available in the default `editor` role.
+- At least two groups in GitLab with users assigned. In our examples below,
+  we'll assume a group named `company` with two subgoups, `admin` and `dev`.
+- Teleport role with access to maintaining `oidc` resources.  This is available
+  in the default `editor` role.
 
 (!docs/pages/includes/commercial-prereqs-tabs.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
 
-## Enable default OIDC authentication
-
-(!docs/pages/includes/enterprise/oidcauthentication.mdx!)
-
 ## Configure GitLab
 
-You should have at least one group configured in GitLab to map to Teleport roles. In this example we use the names `gitlab-dev` and `gitlab-admin`.  Assign users to each of these groups.
-
-1. Create a Application in one of your Groups that will allow using GitLab as a OAuth provider to Teleport.
+You should have at least one group configured in GitLab to map to Teleport roles.
+In this example we use the names `gitlab-dev` and `gitlab-admin`.  Assign users
+to each of these groups.
 
-Settings
+1. Create a Application in one of your Groups that will allow using GitLab as a
+   OAuth provider to Teleport.
 
-- Redirect URL `https://<proxy url>/v1/webapi/oidc/callback` such as `https://teleport.example.com:3080/v1/webapi/oidc/callback`
-- Check `Confidential`, `openid`, `profile`, and `email`.
+   Settings:
 
-![Create App](../../../img/sso/gitlab/gitlab-oidc-0.png)
+   - Redirect URL: `https://<proxy url>/v1/webapi/oidc/callback`.
+   - Check `Confidential`, `openid`, `profile`, and `email`.
 
-2. Collect the `Application ID` and `Secret` in the Application
+   ![Create App](../../../img/sso/gitlab/gitlab-oidc-0.png)
 
-These will be used in the Teleport OIDC Auth Connector.
+2. Collect the `Application ID` and `Secret` in the Application. These will be
+   used in the Teleport OIDC Auth Connector:
```

Comment:
```suggestion
   used in the Teleport OIDC auth connector:
```

For consistency with the rest of the docs, since we tend not to capitalize "auth connector".